desktop-themes/TraditionalOk/gtk-3.0/Makefile.am

```diff
@@ -3,12 +3,7 @@ THEME_NAME=TraditionalOk
 themedir = $(datadir)/themes/$(THEME_NAME)/gtk-3.0
 
 theme_DATA = \
-	applications.css \
-	gtk.css \
-	gtk-widgets.css \
-	gtk-widgets-img.css \
-	mate-applications.css \
-	other-applications.css \
+	gtk.css \ 
```

Comment:
There's a syntax error in this line. Currently it terminates in a whitespace, but it shouldn't
```diff
diff --git a/desktop-themes/TraditionalOk/gtk-3.0/Makefile.am b/desktop-themes/TraditionalOk/gtk-3.0/Makefile.am
index b4c56e30..030a1852 100644
--- a/desktop-themes/TraditionalOk/gtk-3.0/Makefile.am
+++ b/desktop-themes/TraditionalOk/gtk-3.0/Makefile.am
@@ -3,7 +3,7 @@ THEME_NAME=TraditionalOk
 themedir = $(datadir)/themes/$(THEME_NAME)/gtk-3.0
 
 theme_DATA = \
-       gtk.css \ 
+       gtk.css \
        settings.ini
 
 %.ini: %.ini.in
```